wrangler-core/src/test/java/io/cdap/wrangler/TestingRig.java

```diff
@@ -89,7 +89,7 @@ public static List<Row> execute(String[] recipe, List<Row> rows, ExecutorContext
 
     String migrate = new MigrateToV2(recipe).migrate();
     RecipeParser parser = new GrammarBasedParser(Contexts.SYSTEM, migrate, registry);
-    return new RecipePipelineExecutor(parser, context).execute(rows);
+    return new RecipePipelineExecutor(parser, context, null).execute(rows);
```

Comment:
should not pass null for anything that is not annotated as `@Nullable`. Also should normally not pass null for any collection and instead pass an empty collection.

Reply:
Done